src/assets/build/subtract.icon.tsx

```diff
@@ -0,0 +1,12 @@
+export const SubtractIcon = (props: any) => (
```

Comment:
I'm not sure about this name. Is it a subtract?

Reply:
Sounds sensible, when I copied it from figma it was called subtract, but OpenSourceIcon is a better name